tests/cairo_files/instructions/test_environmental_information.cairo

```diff
@@ -76,3 +77,33 @@ func test__exec_address__should_push_address_to_stack{
     assert index0 = Uint256(420, 0);
     return ();
 }
+
+@external
+func test__exec_extcodecopy__{
+    syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr, bitwise_ptr: BitwiseBuiltin*
+}() {
+    // a nonsense test just to get started ;)
+    // Given
+    alloc_locals;
+    
+    // TODO need to initialize/mock a registry contract
+    registry_address.write(1);
+
+    let (bytecode) = alloc();    
+    let stack: model.Stack* = Stack.init();
+    let stack: model.Stack* = Stack.push(stack, Uint256(1, 2)); // size
+    let stack: model.Stack* = Stack.push(stack, Uint256(1, 0)); // offset
+    let stack: model.Stack* = Stack.push(stack, Uint256(2, 0)); // dest_offset
+    let stack: model.Stack* = Stack.push(stack, Uint256(3, 0)); // address
+
+    let ctx: model.ExecutionContext* = TestHelpers.init_context_with_stack(0, bytecode, stack);
+
+
+    // When
+    let result = EnvironmentalInformation.exec_extcodecopy(ctx);
```

Comment:
What are we testing here?
We'd need an integration test to:
Deploy a contract through Kakarot -> Execute extcodecopy on it

Reply:
@Eikix i made this test to stub out a test.

I understand now that a minimal unit test will need to deploy an account registry. Working on that now!